src/app/components/AddEvaluationForm/AddEvaluationForm.tsx

```diff
@@ -92,8 +105,9 @@ export default function AddEvaluationForm({
           evaluations.map((evaluation) =>
```

Comment:
The whole part of the function from here looks very nested with many conditions, is there a way you could make the code a little clearer to read?